### PR TITLE
fix: add testnet-tds to EndpointConfig map

### PR DESCRIPTION
**Pull request by danpaul000**
Fixes https://github.com/solana-labs/solana/issues/4989

### CODE REVIEW

**Comment by mvines** [comment]
You need to use semantic-release commit formatting in this repo, that's why Travis is mad.  
eg:
```
fix: add testnet-tds to EndpointConfig map
```

**Comment by danpaul000** [comment]
@mvines anything else needed now that I've appeased Travis?

File: src/EndpointConfig.js [hunk 1]
```diff
@@ -7,11 +7,13 @@ const urlMap = {
   'testnet-edge': 'http://edge.testnet.solana.com:8899',
   'testnet-beta': 'http://beta.testnet.solana.com:8899',
   testnet: 'http://testnet.solana.com:8899',
+  'testnet-tds': 'http://tds.solana.com:8899',
```
Comment (mvines) [inline src/EndpointConfig.js]: wanna just call this `tds` (drop the `testnet-` prefix)

**Comment by mvines** [comment]
> @mvines anything else needed now that I've appeased Travis?

Nerp, our mission is simply to keep all bots happy.

File: src/EndpointConfig.js (hunk 1 above)
Comment (danpaul000) [reply src/EndpointConfig.js]: Should I style it `TdS` in that case?

Comment (mvines) [reply src/EndpointConfig.js]: lowercase for now, we also use this value to index into metrics.  We prolly need to add a friendly name field soon...

**Pull request merged**